File: python/src/Behavioral/Visitor/TitleBlurbVisitor.py
#//TitleBlurbVisitor - the abstract Visitor
class TitleBlurbVisitor:

	#def __init__():pass

	titleBlurb   = None

	def setTitleBlurb (self,blurbIn) :
		self.titleBlurb = blurbIn

	def getTitleBlurb (self): 
		return self.titleBlurb

	def visit (self,info) :
		raise Exception("Unimplemented method TitleBlurbVisitor.visit")

File: ruby/src/Behavioral/Visitor/TitleBlurbVisitor.rb
#//TitleBlurbVisitor - the abstract Visitor
class TitleBlurbVisitor

	attr_accessor :titleBlurb

	def setTitleBlurb (blurbIn)
 		@titleBlurb = blurbIn
	end

	def getTitleBlurb ()
		return @titleBlurb
	end

	def visit (info)
 		raise "Unimplemented method TitleBlurbVisitor.visit"
	end
end
